src/lib/FlightTasks/tasks/Orbit/FlightTaskOrbit.hpp

```diff
@@ -46,6 +46,13 @@
 #include <uORB/topics/orbit_status.h>
 #include <StraightLine.hpp>
 
+enum class YawBehavior : int {
```

Comment:
@dayjaby commit https://github.com/mavlink/mavlink/pull/1266/commits/84b5a8fd11805072de0cf804d679260890583a15 basically excludes the need for this enumeration.

Reply:
Wouldn't it be better to not have any mavlink include dependencies within flight_tasks? I'd rather follow the same approach as in https://github.com/PX4/Firmware/blob/master/src/lib/flight_tasks/tasks/Utility/ObstacleAvoidance.cpp#L115, where you store the mavlink enum in an uorb msg, for example in orbit_status.msg

Reply:
> Wouldn't it be better to not have any mavlink include dependencies within flight_tasks? I'd rather follow the same approach as in https://github.com/PX4/Firmware/blob/master/src/lib/flight_tasks/tasks/Utility/ObstacleAvoidance.cpp#L115, where you store the mavlink enum in an uorb msg, for example in orbit_status.msg

There's no need for that if the enumeration is defined on the protocol.